### DISCUSSION TITLE
Export SQL without DB Prefix

### DISCUSSION THREAD

**Discussion by mvenghaus** [dbeaver/dbeaver#16817] [Q&A]
Hello,

is there a possibility to export sql without db prefix?

Regards
Marcus

**Comment by LonwoLonwo** [comment]
Hello @mvenghaus 
If I understood you right, you need to enable this data transfer setting:

![2022-04-19 15_36_56-DBeaver Ultimate 22 0 3 - actor](https://user-images.githubusercontent.com/45152336/164005896-66972f24-1291-4acb-b274-77f572a11fde.png)

**Reply by mvenghaus** [reply]
that's it .. thank you!

**Reply by mvenghaus** [reply]
@LonwoLonwo is there also a setting for copy as SQL?

**Reply by LonwoLonwo** [reply]
This one?

![2022-06-17 08_15_30-](https://user-images.githubusercontent.com/45152336/174229963-9d829ed7-0c6c-414f-aef1-0f42262d9c4f.png)

**Reply by mvenghaus** [reply]
yes .. this one without db name in generated sql!?

**Reply by LonwoLonwo** [reply]
You can configure it here:

![2022-06-17 08_55_39-DBeaver 22 1 0 - Playlist](https://user-images.githubusercontent.com/45152336/174234416-6774c9c2-54e4-4c16-aadc-b3ad2268fd27.png)

**Reply by mvenghaus** [reply]
I'm so blind :) .. you're life saver .. THANK YOU
